tests/gas_subsidies/registry_update_test.go

```diff
@@ -36,7 +36,7 @@ import (
 func TestRegistryUpdate_UpdateContract_SponsoredTransactionsCanBePerformed(t *testing.T) {
 
 	// Enable allegro to support setCode transactions.
```

Comment:
Comment is slightly confusing, 

```suggestion
	// Enable a revision with allegro to support setCode transactions.
```

Regarding whenever updating this or not. I think it makes sense to have it updated to test interaction with new hardforks.